### PR TITLE
Add Health and Safety Information

### PR DESCRIPTION
**Pull request by BreadFish64**
Adds Health and Safety Information.

*Automatically generated by PR builder*

### CODE REVIEW

File: health-and-safety-information/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Health and Safety Information"
+description = "Health and safety information for the Nintendo 3DS is pre-installed on the 3DS as a system application,"
```
Comment (TheKoopaKingdom) [inline health-and-safety-information/game.dat]: That last `,` should be a `.`

File: health-and-safety-information/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs_shared_font = true
+
+[[releases]]
+title = "0004001000021300"
+region = "usa"
+release_date = "2011-03-27"
+
+[[ testcases ]]
+ title = "0004001000021300"
+ compatibility = "1"
```
Comment (TheKoopaKingdom) [inline health-and-safety-information/game.dat]: If this [has no issues](https://github.com/citra-emu/citra-games-wiki/wiki/Health-and-Safety-Information), the `compatibility` should be `0` for `Perfect`.

Comment (BreadFish64) [inline health-and-safety-information/game.dat]: I wasn't sure if needing the system font was considered a workaround so I'll change that.

**Comment by BreadFish64** [comment]
I see what you mean about the folder capitalization now; it was automatically generated and I'm not entirely sure how to change it.

**Comment by TheKoopaKingdom** [comment]
I don't think you renamed the folder correctly, the boxart and icon are using the old path.

**Comment by BreadFish64** [comment]
Yeah, I don't think I did it correctly either, trying to figure out how to change the path on the images rn

**Comment by BreadFish64** [comment]
woops, need to change image names

**Comment by BreadFish64** [comment]
Well that was probably the least efficient way to do that but it should work

**Comment by TheKoopaKingdom** [comment]
LGTM (Again :p).

**Pull request merged**